db/schema.rb

```diff
@@ -123,8 +123,8 @@
   create_table "distributors_shipping_methods", :force => true do |t|
     t.integer  "distributor_id"
     t.integer  "shipping_method_id"
-    t.datetime "created_at",         :null => false
-    t.datetime "updated_at",         :null => false
+    t.datetime "created_at"
```

Comment:
Why does the schema change if there are no migrations in this commit?

Reply:
Thank God you pointed this out! This one is especially dangerous